src/test/java/com/cloudbees/jenkins/support/configfiles/OtherConfigFilesComponentTest.java

```diff
@@ -0,0 +1,78 @@
+package com.cloudbees.jenkins.support.configfiles;
+
+import hudson.util.Secret;
+import org.apache.commons.io.FileUtils;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import java.io.File;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by valentina on 28/10/16.
+ */
+public class OtherConfigFilesComponentTest {
+
+    @Rule
+    public JenkinsRule r = new JenkinsRule();
+
+    private String xml;
+
+    private final String expectedXml = "<com.cloudbees.plugins.credentials.SystemCredentialsProvider plugin=\"credentials@1.18\">\n" +
```

Comment:
cause I am comparing strings eventually, one being dynamic since I generate the secret on runtime. It's an overkill  IMHO